feature/bgp/policybase/otg_tests/link_bandwidth_test/link_bandwidth_test.go

```diff
@@ -719,7 +837,11 @@ func configureExtCommunityRoutingPolicy(t *testing.T, dut *ondatra.DUTDevice) {
 			t.Fatalf("Unsupported vendor %s for native cmd support for deviation 'BgpDeleteLinkBandwidthUnsupported'", dut.Vendor())
 		}
 		helpers.GnmiCLIConfig(t, dut, delLinkbwCLIConfig)
+	} else {
+		gnmi.Update(t, dut, gnmi.OC().RoutingPolicy().Config(), rpDelLinkbw)
 	}
+
+	fptest.LogQuery(t, "", gnmi.OC().RoutingPolicy().Config(), root)
```

Comment:
Export policy mixup & failure has been resolved, but do we need to print here routing-policy configuration on console? If not can we remove this.